profiles/debug.lua

```diff
@@ -43,14 +43,16 @@ mode = {
 
 -- input tags, normally extracted from OSM data
 local way = {
-  highway = 'primary',
```

Comment:
i will clean up the debug.lua file, so it's just the general lua debugging tool, and then do my local work in a git ignore file. no need to have these kind of development changes in the commits going forward